manager/manager/utils.py

```diff
@@ -720,3 +781,243 @@ def assert_time_data(time_data):
     if not isinstance(time_data["tai_to_utc"], float):
         return False
     return True
+
+
+def send_smtp_email(to, subject, html_content, plain_content):
+    """Send an email using the SMTP protocol.
+
+    Parameters
+    ----------
+    to : `str`
+        The email address of the recipient
+    subject : `str`
+        The subject of the email
+    html_content : `str`
+        The content of the email in HTML format
+    plain_content : `str`
+        The content of the email in plain text format
+
+    Notes
+    -----
+    The following environment variables are required to send the email:
+    - SMTP_USER: The SMTP user name
+    - SMTP_PASSWORD: The SMTP user password
+
+    Returns
+    -------
+    bool
+        True if the email was sent successfully, False if not
+    """
+
+    try:
+        # Create message container - the correct MIME type
+        # is multipart/alternative.
+        msg = MIMEMultipart("alternative")
+        part1 = MIMEText(plain_content, "plain")
+        part2 = MIMEText(html_content, "html")
+
+        # Attach parts into message container.
+        # According to RFC 2046, the last part of
+        # a multipart message, in this case
+        # the HTML message, is best and preferred.
+        msg.attach(part1)
+        msg.attach(part2)
+
+        msg["Subject"] = subject
+        msg["From"] = f"{os.environ.get('SMTP_USER')}@lsst.org"
```

Comment:
In the CHANGELOG you refer to the `mail.lsst.org` domain but here you correctly use `lsst.org`. Please update the CHANGELOG to reflect that.

Also, what happens if the `SMTP_USER` variable already contains the domain? Maybe add a check to make sure that you need to add the domain?